tools/inference_gradio.py

```diff
@@ -0,0 +1,126 @@
+"""Anomalib Gradio Script.
+
+This script provide a gradio web interface
+"""
+
+from argparse import ArgumentParser, Namespace
+from importlib import import_module
+from pathlib import Path
+from typing import Tuple, Union
+
+import gradio as gr
+import gradio.inputs
+import gradio.outputs
+import numpy as np
+from omegaconf import DictConfig, ListConfig
+from skimage.segmentation import mark_boundaries
+
+from anomalib.config import get_configurable_parameters
+from anomalib.deploy.inferencers.base import Inferencer
+from anomalib.post_processing import compute_mask, superimpose_anomaly_map
+
+
+def infer(
+    image: np.ndarray, inferencer: Inferencer, threshold: float = 50.0
+) -> Tuple[np.ndarray, float, np.ndarray, np.ndarray, np.ndarray]:
+    """Inference fonction, return anomaly map, score, heat map, prediction mask ans visualisation.
```

Comment:
Minor issue but there is a typo at `function`.
Also, we are using Google format as our documentation style. So instead of 
```
:param image: image
:type image: np.ndarray
```
It should be
```
image: (np.ndarray): Image
```